sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -155,6 +159,67 @@ dif_rstmgr_result_t dif_rstmgr_reset_info_clear(const dif_rstmgr_t *handle) {
   return kDifRstmgrOk;
 }
 
+dif_rstmgr_result_t dif_rstmgr_alert_info_set_enabled(
+    const dif_rstmgr_t *handle, dif_rstmgr_toggle_t state) {
+  if (handle == NULL) {
+    return kDifRstmgrBadArg;
+  }
+
+  mmio_region_t base_addr = handle->params.base_addr;
```

Comment:
Could you just inline this? If you really want to keep it, move it to the site of first use.